test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -8317,26 +8317,6 @@ BOOST_AUTO_TEST_CASE(using_for_by_name)
 	ABI_CHECK(callContractFunction("x()"), encodeArgs(u256(6 * 7)));
 }
 
-BOOST_AUTO_TEST_CASE(bound_function_in_var)
-{
-	char const* sourceCode = R"(
-		library D { struct s { uint a; } function mul(s storage self, uint x) returns (uint) { return self.a *= x; } }
-		contract C {
-			using D for D.s;
-			D.s public x;
-			function f(uint a) returns (uint) {
-				x.a = 6;
-				var g = x.mul;
-				return g({x: a});
```

Comment:
It seems you can use `return (x.mul)({x: a});` here, which should basically test the same things.